tutorials/single_node_tutorial/single_gpu_tutorial.ipynb

```diff
@@ -91,42 +91,48 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "id": "8add9bbd",
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com\n",
-      "Requirement already satisfied: jsonlines in /usr/local/lib/python3.10/dist-packages (4.0.0)\n",
-      "Requirement already satisfied: attrs>=19.2.0 in /usr/local/lib/python3.10/dist-packages (from jsonlines) (23.2.0)\n",
-      "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
-      "\u001b[0m"
-     ]
-    }
-   ],
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
    "source": [
     "!pip install jsonlines"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
+   "id": "97079227-d9c3-40d2-b939-64b221b86990",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!export DASK_DATAFRAME__QUERY_PLANNING=False"
```

Comment:
This should be changed to
```
%env DASK_DATAFRAME__QUERY_PLANNING False
```

And after doing that you should be able to remove the line below too:
```python
os.environ['DASK_DATAFRAME__QUERY_PLANNING'] = 'False'
```

Reply:
@ryantwolf Hi Ryan, fix updated.

Reply:
@nicoleeeluo can you delete the code instead of commenting it out?

Reply:
@ryantwolf Thanks for pointing this out! I have deleted them